feathr_project/setup.py

```diff
@@ -35,6 +35,7 @@
         "pyyaml",
         "Jinja2",
         "tqdm",
+        "pyarrow",
```

Comment:
Let's put test deps into: https://stackoverflow.com/questions/15422527/best-practices-how-do-you-list-required-dependencies-in-your-setup-py.